roles/cds_for_icpc/files/deploy_backup_tar

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/python3
+
+from os import listdir, chdir, mkdir
+from os.path import isfile, islink, getmtime, join, realpath, isdir
+import tarfile
+
+backup_file = "backups.tar"
+
+if isfile(backup_file):
+    path = "/srv/cds/contests/cds/"
+    destdir = join(realpath(path), "teams")
+    if not isdir(destdir):
+        mkdir(destdir)
+    with tarfile.open(backup_file, "r") as tar:
+        tar.extractall(destdir)
+        print("extracted {} to {}".format(backup_file, destdir))
+else:
+    print("ERROR: {} does not exist".format(backup_file))
```

Comment:
Assuming a reasonable version of python3 you can now do this
```suggestion
    print(f"ERROR: {backup_file} does not exist")
```

also on line 16, I find it easier to read but I'm not sure what is your opinion

Reply:
ah yes,  this script was originally used on python3.6.   I have changed it to use f-strings.